testsuite/pytests/test_aeif_cond_beta_multisynapse.py

```diff
@@ -0,0 +1,418 @@
+        ts = vm.events["times"]
+        Vms = vm.events["V_m"]
+
+        if have_plotting:
+            # plot timeseries as a sanity check
+            import matplotlib.pyplot as plt
+
+            fig, ax = plt.subplots(nrows=6)
+            ax[0].plot(ts, Vms, label="V_m multisyn", alpha=0.5)
+
+        V_m_summed = 0.0
+
+        # loop over synapse index
+        for i in range(3):
+            t0 = spike_time + delays[i]  # peak starting time
+
+            # Evaluates the driving force
+            Vforce = E_rev[i] - Vrest
+
+            # coefficient for approximate linear relationship between V(t) and g(t)
+            coeff = weights[i] * Vforce / g_L
+
+            # peak time
+            t_p = tau_decay[i] * tau_rise[i] / (tau_decay[i] - tau_rise[i]) * np.log(tau_decay[i] / tau_rise[i])
+
+            # normalization coefficient for synaptic conductance
+            g0 = coeff / (np.exp(-t_p / tau_decay[i]) - np.exp(-t_p / tau_rise[i]))
+
+            # approximate analytical calculation of V(t)
+            Vtheor = g0 * (np.exp(-(ts - t0) / tau_decay[i]) - np.exp(-(ts - t0) / tau_rise[i]))
+            Vtheor[ts < t0] = 0.0
+            V_m_summed += Vtheor
+            Vtheor += Vrest
+
+            if have_plotting:
+                ax[i + 1].plot(ts, Vtheor)
+
+                for _ax in ax:
+                    _ax.legend()
+
+        V_m_summed += Vrest
+
+        if have_plotting:
+            ax[0].plot(ts, V_m_summed, label="summed")
+
+            ax[-1].semilogy(ts, np.abs(Vms - V_m_summed), label="error")
+            fname = "test_aeif_cond_beta_multisynapse.png"
+            fname = os.path.join(os.environ.get("REPORTDIR", ""), fname)
+            fig.savefig(fname)
+
+        # large testing tolerance due to approximation (see documentation of the test)
+        np.testing.assert_allclose(Vms, V_m_summed, atol=0.0, rtol=1e-5)
+        np.testing.assert_allclose(Vms, V_m_summed, atol=1e-3)
+
+    @pytest.mark.parametrize("t_ref", [0.0, 0.1])
+    def test_refractoriness_clamping(self, t_ref):
+        r"""Test that refractoriness mechanism and voltage reset/clamping after spike works for t_ref == 0"""
+        nest.ResetKernel()
+        nest.resolution = 0.1
+        V_reset = -111.0
+
+        nrn = nest.Create(
+            "aeif_cond_beta_multisynapse",
+            params={"w": 0.0, "a": 0.0, "b": 0.0, "Delta_T": 0.0, "t_ref": t_ref, "I_e": 1000.0, "V_reset": V_reset},
+        )
+
+        sr = nest.Create("spike_recorder", params={"time_in_steps": True})
+        vm = nest.Create("voltmeter", params={"time_in_steps": True, "interval": nest.resolution})
+
+        nest.Connect(nrn, sr)
+        nest.Connect(vm, nrn)
+
+        nest.Simulate(10.0)
+
+        stime = sr.events["times"][0] - 1  # minus one because of 1-based indexing
+
+        # test that V_m == V_reset at spike time
+        nptest.assert_almost_equal(vm.events["V_m"][stime], V_reset)
```

Comment:
I accepted a suggestion here too quickly. Needs to be reverted:
```suggestion
        np.testing.assert_almost_equal(vm.events["V_m"][stime], V_reset)
```